README.md

```diff
@@ -466,6 +466,60 @@ Simbody is installed into the `usr/` directory.  The directory
     * `examples/src` source code for the examples.
     * `examples/bin` symbolic link to executable examples.
 
+Windows using MinGW
+-------------------
+
+Warning: The [MinGW](http://sourceforge.net/projects/mingw-w64/)
+generation and build is experimental!
+
+This build is still experimental, because of the precompiled libraries
+Simbody depends on (Blas and Lapack).
+To ease the compilation Windows, Simbody provides precompiled libraries for Blas and Lapack.
+
+* On Windows 32 Bits, these were compiled with a Dwarf exception mechanism,
+* On Windows 64 Bits, these were compiled with a SJLJ exception mechanism.
+
+If one chooses a MinGW compilation, we need to respect these exception mechanism.
+A program can not rely on both mechanisms.
+This means that if we want to use the precompiled libraries, our MinGW installation should
+have the same exception mechanism.
+Otherwise, we need to provide our own Blas and Lapack libraries.
+
+To see which exception mechanism is used, user can have a look at dlls located in the `bin` directory of MinGW.
+The name of mechanism is present in the file `libgcc_XXXX.dll`, where `XXXX` can be `dw`, `seh` or `sljl`.
+On some MinGW version, this information is also available by looking at the result of `gcc --version`.
```

Comment:
"user can have a look" -> "the user can look"

"On some MinGW version" -> "For some MinGW versions"